firefox-ios/Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -192,7 +192,7 @@ class SearchSettingsTableViewController: ThemedTableViewController, FeatureFlagg
                     prefs: profile.prefs,
                     theme: themeManager.currentTheme,
                     prefKey: PrefsKeys.FirefoxSuggestShowNonSponsoredSuggestions,
-                    defaultValue: profile.prefs.boolForKey(PrefsKeys.FirefoxSuggestShowNonSponsoredSuggestions) ?? true,
+                    defaultValue: profile.prefs.boolForKey(PrefsKeys.FirefoxSuggestShowNonSponsoredSuggestions) ?? false,
```

Comment:
hey @cyndichin, realized the default was wrong here since the sponsored url filtering relies on the pref. the initial state of the toggle is "on" but the pref is not set.

Reply:
Nice catch! 🙏🏼

Reply:
thanks @tiftran ! when I pull down the branch it seems the logic is reversed for the toggle now. Seems I see the firefox suggest + sponsored items when the toggles are OFF.

https://github.com/mozilla-mobile/firefox-ios/assets/6743397/f5551f92-9c81-4c55-b230-88ce2e4a4b77